Resources/Prototypes/Entities/Objects/Specific/Medical/healing.yml

```diff
@@ -84,7 +84,7 @@
     damageContainer: Biological
     damage:
       types:
-        bloodloss: -0.5 #minor bleed rate reduction.
+        Bloodloss: -0.5 #minor bleed rate reduction.
```

Comment:
A minor typo on my part. Can you also edit the comment to say "bloodloss damage" it occured to me I never fixed that when I changed it from reducing bleed rate reducing bloodloss damage apparently...

Also make sure bloodpacks can still be used if the person does not have bloodloss damage because their main use is to refill their blood level-

Reply:
If I can find a good way to test that I will do that later

Reply:
You can debug>edit solutions and turn to the bloodstream section to remove blood which is the easiest way. Itll remove all their blood without applying any brute damage. For testing bloodloss issues you could also add dex+ to the chemstream so the test mob is always healing off bloodloss damage. It shouldnt be a big issue in lives games iirc because the only time youd need blood is if youre taking damage from having low blood level anyways. Im just thinking there might be a case where someone has low blood levels and they cant restore blood because they havent taken any bloodloss damage-

Reply:
Getting the amount of blood in a person is turning out to be harder than I thought